src/ol/dom.js

```diff
@@ -19,22 +19,24 @@ export function createCanvasContext2D(
   opt_canvasPool,
   opt_Context2DSettings
 ) {
-  const canvas =
+  const canvas = /** @type {HTMLCanvasElement} */ (
     opt_canvasPool && opt_canvasPool.length
       ? opt_canvasPool.shift()
       : WORKER_OFFSCREEN_CANVAS
       ? new OffscreenCanvas(opt_width || 300, opt_height || 300)
-      : document.createElement('canvas');
+      : document.createElement('canvas')
+  );
+  if (canvas.style) {
+    canvas.style.all = 'initial';
+  }
```

Comment:
With the type cast you added above, it should not be necessary to check for `canvas.style` beforehand:

```suggestion
  canvas.style.all = 'initial';
```